controllers/dspipeline_controller.go

```diff
@@ -271,24 +280,20 @@ func (r *DSPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 
 	r.PublishMetrics(
 		dspa,
+		util.GetConditionByType(config.DatabaseAvailable, conditions),
+		util.GetConditionByType(config.ObjectStoreAvailable, conditions),
 		util.GetConditionByType(config.APIServerReady, conditions),
 		util.GetConditionByType(config.PersistenceAgentReady, conditions),
 		util.GetConditionByType(config.ScheduledWorkflowReady, conditions),
 		util.GetConditionByType(config.CrReady, conditions),
 	)
-
 	return ctrl.Result{}, nil
 }
 
 // handleReadyCondition evaluates if condition with "name" is in condition of type "conditionType".
 // this procedure is valid only for conditions with bool status type, for conditions of non bool type
 // results are undefined.
-func (r *DSPAReconciler) handleReadyCondition(
-	ctx context.Context,
-	dspa *dspav1alpha1.DataSciencePipelinesApplication,
-	name string,
-	condition string,
-) (metav1.Condition, error) {
+func (r *DSPAReconciler) handleReadyCondition(ctx context.Context, dspa *dspav1alpha1.DataSciencePipelinesApplication, name string, condition string) (metav1.Condition, error) {
```

Comment:
how come you made this into one line? looks cleaner the other way no?

Reply:
linter yelled at me :)